packages/x-date-pickers/src/MobileDatePicker/MobileDatePicker.test.tsx

```diff
@@ -22,29 +22,6 @@ const WrappedMobileDatePicker = withPickerControls(MobileDatePicker)({
 describe('<MobileDatePicker />', () => {
   const { clock, render } = createPickerRenderer({ clock: 'fake', clockConfig: new Date() });
 
-  it('selects the closest enabled date if selected date is disabled', () => {
```

Comment:
This test is still passing

https://codesandbox.io/s/basicdatepicker-material-demo-forked-kegr9m?file=/demo.tsx

Reply:
Yes, I just moved it lower in the component tree to not just test it on `MobileDatePicker` :+1:

Reply:
My bad, just found it in the Calendar tests